specification/billing/resource-manager/Microsoft.Billing/stable/2023-04-01/customer.json

```diff
@@ -1,72 +1,80 @@
 {
   "swagger": "2.0",
   "info": {
-    "description": "Billing client provides access to billing resources for Azure subscriptions.",
     "title": "BillingManagementClient",
+    "description": "Documentation for Microsoft.Billing.",
+    "contact": {
+      "name": "Microsoft.Billing",
+      "url": "https://learn.microsoft.com/en-us/rest/api/billing/",
+      "email": "cabbpt@microsoft.com"
+    },
     "version": "2023-04-01"
   },
-  "host": "management.azure.com",
-  "schemes": [
-    "https"
-  ],
-  "consumes": [
-    "application/json"
-  ],
-  "produces": [
-    "application/json"
-  ],
   "paths": {
     "/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/billingProfiles/{billingProfileName}/customers": {
       "get": {
```

Comment:
I think we should just keep the top-level "Customers" resource type under billing account... Then when someone wants to query for customers under a specific billing profile, just use a $filter. This avoids ambiguity and problems around having multiple containers for the same resource type.

Reply:
@ramkd12 @amberbhargava to comment here

Reply:
We need to keep billing scope resources in a "hierarchical" pattern to support migration from CPAS to PAS in the future. It is intentional to put customers at billing profile scope for this reason.

Reply:
We can still have a top-level query at the billing account scopes.